config/magpie.ini

```diff
@@ -32,7 +32,7 @@ magpie.max_restart = 5
 # This secret should be the same in Twitcher !
 magpie.secret =
 magpie.push_phoenix = true
-magpie.config_path =
+# magpie.config_path =
```

Comment:
The config is also used for other stuff (see [config.yaml](https://github.com/Ouranosinc/Magpie/blob/d3937036c09b5609cfa39c87ccc1a2b1e94538aa/config/config.yml)) such as defining providers and permissions, explaining the more generic name.

It was commented because else, the environment variable MAGPIE_TEST_USER_WEBHOOK_URL defined in magpie.env would be overriden by this empty value defined in magpie.ini, which would make the webhook tests fail.

 I talked about this with @fmigneault and from what I understood, the .ini file is mostly for demo purposes. If a user changes that value, it would be to run the app locally and not to run the tests. So we leave it commented to let the tests run by default, but if a user wants to run the app to try a different config, he can change it in this .ini file.
 
 I guess I could add a small comment here though such as :
 # This value can be defined to run the app locally with a specific config. Note that this will break the webhook tests, that use another config_path, by overriding the environment variable.
 
 But maybe there is a cleaner way.

Reply:
Indeed the comment could help to better understand why the config is commented out. I will let @fmigneault evaluate if there is a better way to describe the situation.

Reply:
Commenting this value is ok as the default in `magpie.constants` is to set it to `None`, and if overridden by anything, it takes priority. So someone providing this value should do it explicitly. 

I feel the webhook test shouldn't depend on INI config regardless though. When the TestApp is created, you can pass it a settings dict with enforced config values. You can provide the desired webhook URL this way. 
I would avoid using `MAGPIE_TEST_USER_WEBHOOK_URL`. Only the webhook test will use this value, so it should be defined only there.